test/karma.conf.cjs

```diff
@@ -1,3 +1,6 @@
+const dotenv = require('dotenv');
+dotenv.config();
```

Comment:
The PR doesn't show this, but needed to change the `FIREFOX_BIN` env var to the path to firefox-esr on linux. I did this by setting the env var in a new .env file at the root of the project. This just reads it and ensures the var is set before tests run.